tsl/test/expected/continuous_aggs_query-10.out

```diff
@@ -49,8 +49,16 @@ from conditions
 group by time_bucket('1day', timec), location;
 NOTICE:  adding index _materialized_hypertable_2_location_timec_idx ON _timescaledb_internal._materialized_hypertable_2 USING BTREE(location, timec)
 SET timescaledb.current_timestamp_mock = '2018-12-31 00:00';
+--compute time_bucketted max+bucket_width for the materialized view
+SELECT time_bucket('1day' , q.timeval+ '1day'::interval)
+FROM ( select max(timec)as timeval from conditions ) as q; 
+         time_bucket          
+------------------------------
+ Sat Nov 03 17:00:00 2018 PDT
+(1 row)
+
 REFRESH MATERIALIZED VIEW mat_m1;
-LOG:  materializing continuous aggregate public.mat_m1: nothing to invalidate, new range up to Fri Dec 28 16:00:00 2018 PST
+LOG:  materializing continuous aggregate public.mat_m1: nothing to invalidate, new range up to Sat Nov 03 17:00:00 2018 PDT
```

Comment:
Is there good explanation why it was PST before the change?

Reply:
daylight savings time - switches between PST and PDT.

Reply:
I understand this, but I don't understand why Dec 28 was PST, not PDT.